### PR TITLE
Group parameters for RDSH compound templates

### PR DESCRIPTION
**Pull request by dvu70**
Create dualaz template
Update tripleaz and singlez templates

### CODE REVIEW

File: templates/ra_dualaz_rdsh_elb.compound [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+            "Properties" :
+            {
+                "TemplateURL" : "https://s3.amazonaws.com/app-chemistry/templates/ra_rdsh_security_group.element",
+                "Parameters" :
+                {
+                    "VPC" : { "Ref" : "VPC" },
+                    "UseELB" : "true"
+                }
+            }
+      	},
```
Comment (lorengordon) [inline templates/ra_dualaz_rdsh_elb.compound]: Bad whitespace alignment

File: templates/ra_dualaz_rdsh_elb.compound [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+            "Properties" :
+            {
+                "TemplateURL" : "https://s3.amazonaws.com/app-chemistry/templates/ra_rdsh_elb.element",
+                "Parameters" :
+                {
+                    "PrivateSubnetIDs" : { "Fn::Join" : [ ",", { "Ref" : "ElbPrivateSubnetIDs" } ] },
+                    "SecurityGroupId" : { "Fn::GetAtt" : [ "RdshSecurityGroupStack", "Outputs.RdshElbSecurityGroupId" ] }
+                }
+            }
+      	},
```
Comment (lorengordon) [inline templates/ra_dualaz_rdsh_elb.compound]: whitespace alignment

File: templates/ra_dualaz_rdsh_elb.compound [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+                        "Fn::GetAtt" :
+                        [
+                            "RdshSecurityGroupStack",
+                            "Outputs.RdshSecurityGroupId"
+                        ]
+                    },
+                    "VPC" : { "Ref" : "VPC" }
+                }
+            }
+      	},
```
Comment (lorengordon) [inline templates/ra_dualaz_rdsh_elb.compound]: whitespace alignment

**Comment by dvu70** [comment]
Wow eagle eye. Believe i got them all. Fixed for other files also

**Comment by lorengordon** [comment]
lgtm, :shipit:

**Pull request merged**